### ISSUE TITLE
The fit_gradients array is not inizialized in gyration

### ISSUE THREAD

**Issue by giacomofiorin** [colvars/colvars#95]
`gyration` has been using the `b_center` flag to remove the center-of-geometry motion of the group. `fit_gradients` used to be always calculated, but their calculation was later skipped when they are zero by construction (for example, in the minimum RMSD). Due to this behavior, the `fit_gradients` array was not filled for `gyration`, giving rise to a segmentation fault at the first time step.

**Comment by giacomofiorin** [comment]
*Temporary workaround*: using explicitly fitting options, as in:
```colvar {
    name rg
    gyration {
        atoms { ... }
        centerReference yes
        rotateReference no
        refPositionsFile <a set of coordinates centered on the origin>
    }
}```

**Comment by giacomofiorin** [comment]
*Permanent fix*: code is fixed by commit 4dd7548 (2016-12-22)
